health-check/src/uyuni_health_check/config_loader.py

```diff
@@ -0,0 +1,66 @@
+import os
+import configparser
+from pathlib import Path
+import yaml
+import json
+from jinja2 import Environment, FileSystemLoader
+
+class ConfigLoader:
+    def __init__(self):
+        self.base_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
We probably want to have the config outside of the Python package in the future.